src/main/resources/templates/userinfo.mustache

```diff
@@ -137,6 +137,14 @@
                             {{driving_permit_claim_present}}
                         </dd>
                     </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Social Security Record Claim Present
+                        </dt>
+                        <dd class="govuk-summary-list__value" id="user-info-social-security-record-claim">
+                            {{social-security-record_claim_present}}
```

Comment:
The driving permit is all underscores, this mixes underscores and hyphens. Is that correct?